rasa/core/config.py

```diff
@@ -216,7 +216,7 @@ def migrate_mapping_policy_to_rules(
     # finally update the policies
     policies = _drop_policy(MappingPolicy.__name__, policies)
 
-    if new_rules and not has_rule_policy:
+    if not has_rule_policy:
```

Comment:
Do we always want the rule policy?

Reply:
From the last comment of @koaning with the logic of which I agree, yes, we will now always have the `RulePolicy` when we are migrating the `MappingPolicy` (even if we have no rules). From what I have seen this function will only be called if the `MappingPolicy` is present in the `config.yml` file so it doesn’t affect other cases.